src/ProofObjects.lidr

```diff
@@ -2,10 +2,8 @@
 
 > module ProofObjects
 
-\[
-  \say{"\textit{Algorithms are the computational content of proofs.}" - Robert
+\say{"\textit{Algorithms are the computational content of proofs.}" - Robert
```

Comment:
This results in the following in the PDF.

> “”Algorithms are the computational content of proofs.” - Robert Harper”

I'm not married to this exact solution, but I don't like the double quotes as it is now.